solutions/cf-231A.mdx

```diff
@@ -74,4 +74,15 @@ import java.io.*;
 
```

Comment:
can you check that the indentation in the java solution is tabs? indentation looks off on the website
![image](https://user-images.githubusercontent.com/63619830/123561444-31a8ab80-d75d-11eb-8d14-3d9761e4412f.png)

Reply:
fixed